library/ts-rollup/src/graph.ts

```diff
@@ -0,0 +1,51 @@
+import { mxgraph } from './initializer';
```

Comment:
I would have put all the code in the main source file and let the candidate refactor.
And add something in the README explaining that this is just a starting point.
This will also easier for backport in the javascript project.

Reply:
yes
like we have discussed - the refactoring left for the candidate is a good test 👍